docs/reference/analysis/tokenizers/simplepattern-tokenizer.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[analysis-simplepattern-tokenizer]]
+=== Simple Pattern Tokenizer
+
+experimental[]
+
+The `simplepattern` tokenizer uses a regular expression to capture matching
+text as terms. The set of regular expression features it supports is more
+limited than the <<analysis-pattern-tokenizer,`pattern`>> tokenizer, but the
+tokenization is generally faster.
+
+This tokenizer does not support splitting the input on a pattern match, unlike
+the <<analysis-pattern-tokenizer,`pattern`>> tokenizer. To split on pattern
+matches using the same restricted regular expression subset, see the
+<<analysis-simplepatternsplit-tokenizer,`simplepatternsplit`>> tokenizer.
+
+This tokenizer uses http://lucene.apache.org/core//6_5_1/core/org/apache/lucene/util/automaton/RegExp.html[Lucene regular expressions].
+For an explanation of the supported features and syntax, see <<regexp-syntax,Regular Expression Syntax>>.
+
+The default pattern is the empty string, which produces no terms. This
+tokenizer should always be configured with a non-default pattern.
+
+[WARNING]
+.Beware of Pathological Regular Expressions
+========================================
+
+A badly written regular expression could run very slowly or even throw a
+StackOverflowError and cause the node it is running on to exit suddenly.
```

Comment:
I believe this isn't true for this tokenizer. The nice thing about lucene's regexes is that badly written regexes tend to explode up front with "TooComplexToDeterminizeException". That isn't to say that all regexes that don't throw that don't have that problem, just that I don't think you need to be "beware". So I think we can drop this warning entirely from this tokenizer.

In fact, this is the real reason we like this tokenizer. It might be faster than the old pattern one, but the reason we want this is that it is much safer.